packages/p2p/src/components/advertiser-page/advertiser-page-dropdown.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Dropdown, Icon } from '@deriv/components';
+import { localize } from 'Components/i18next';
+import { useStores } from 'Stores';
+import './advertiser-page.scss';
+import { observer } from 'mobx-react-lite';
+
+const AdvertiserPageDropdown = () => {
```

Comment:
Do you think renaming this component to `AdvertiserPageDropdownMenu` would make more sense since it is a dropdown-menu?

Reply:
It does make sense, I did think about using that name initially, but it is quite long already.